src/ruby/lib/grpc/generic/client_stub.rb

```diff
@@ -398,13 +398,12 @@ def server_streamer(method, req, marshal, unmarshal,
     # responses by throwing StopIteration, but can only happen either
     # if bidi_call#writes_done is called.
     #
-    # To terminate the RPC correctly the block:
+    # To terminate the RPC correctly the block, bidi#writes_done must be called
```

Comment:
While this is being updated, I am realizing that this suggestion is very out of date, and bidi#writes_done hasn't existed for a while. Can we remove these comments from the docs?

Can we replace this with:

```
To properly terminate the RPC, the responses should be completely iterated through; one way to do this is to loop on responses#next until no further responses are available.
```

Reply:
@apolcyn Thank you for your review! :sparkles:
I replaced it.